Added initiate.json NOTE:gear type limits should be determined by sec…

mwalla opened the pull request:
added initiate.json
gear has subcategories 'old_fashioned' and 'modern'
'sect' section is exactly like 'organization' for professional
left 'keys' empty cuz i dunno what to do in there

mwalla committed:
Added initiate.json NOTE:gear type limits should be determined by sect attributes, and keys section is empty.

llpk79 reviewed (changes requested):
change "pick" to "limit"

Notice "moves": "9" refers to investigate a mystery. Therefore, "keys": "investigate a mystery" will include ["9"]

Looks like you'll have to work out with the keeper what the sect choices map to as far as moves and add in later if you wish. Make keys:haven -> keys:sect if you do and follow pattern above.

Missing closing parentheses in gear:old_fashioned:3,4 and gear:modern:5

mwalla committed:
made changes per code review:
fixed typos
filled out 'keys' section relating special class move options to one another

llpk79 reviewed (approved):
Perfect
